compose/foundation/foundation/src/commonMain/kotlin/androidx/compose/foundation/text/CoreTextField.kt

```diff
@@ -1191,3 +1193,9 @@ private fun notifyFocusedRect(
 internal const val USE_WINDOW_FOCUS_ENABLED = false
```

Comment:
is this a flag of a kind? Is this supposed to be set externally somehow?

Reply:
To be honest, I don't know. The logic I added is not related (at least directly) by this flag. It was there before.